app/controllers/concerns/umrdr/works_controller_behavior.rb

```diff
@@ -4,6 +4,41 @@ module WorksControllerBehavior
     include Sufia::Controller
     include Sufia::WorksControllerBehavior
     include CurationConcerns::CurationConcernController
+       
+    # override curation concerns, add form fields values
+    def build_form
+    
+      super
+
+      date_range =  @form["date_coverage"].first
+      date_from_to = date_range.split('/')
+
+      start_dt = EDTF.parse(date_from_to[0])
+      end_dt =  EDTF.parse(date_from_to[1])
+     
+      if start_dt.day_precision?
```

Comment:
On a blank form or generic_work/new does this throw a `no method 'day_precision?' for nil` error?